modules/programs/autorandr.nix

```diff
@@ -0,0 +1,200 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.autorandr;
+
+  profileModule = types.submodule {
+    options = {
+      fingerprint = mkOption {
+        type = types.attrsOf types.string;
+        description = ''
+          Output name to EDID mapping.
+          Use <code>autorandr --fingerprint</code> to get current setup values.
+        '';
+        default = {};
+      };
+      config = mkOption {
```

Comment:
I think it looks a bit nicer with an empty line between options. I also then find it easier to jump between options in my text editor (using jump to next empty line).